queries.sql

```diff
@@ -7,4 +7,26 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = 't';
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
- 
\ No newline at end of file
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] There are verification steps that we've left out. If any, I will be pointing them out in the transaction queries that follow this block.

  First off, before ROLLBACK and after, we were supposed to verify that changes were made and rollbacked. Project requirements are very explicit on this, we, therefore, cannot overlook them. See below:
  
  > - Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction.
  
  ## Example
  If we included the verification steps, we would have something like this:
  
  ```sql
  BEGIN; -- start transaction
  
  UPDATE animals SET species = 'unspecified'; -- make change
  
  SELECT species from animals; -- verify that change was made
  
  ROLLBACK;
  
  SELECT species from animals; -- verify that change was undone
  ```
  
  Please, add these verification steps in the query transactions.